xrtpy/response/tests/test_effective_area.py

```diff
@@ -99,7 +101,7 @@ def test_EffectiveArea_exception_is_raised(name, date):
 def get_IDL_data_files():
     filter_data_files = []
     for dir in get_pkg_data_filenames(
-        "data/effective_area_IDL_testing_files", package="xrtpy.response.tests"
```

Comment:
Is the plan to keep the old testing files?

Reply:
No, I plan to remove them soon